apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/AggregateQueryTest.cs

```diff
@@ -53,37 +78,117 @@ public async Task GetSnapshotAsync_VerifySnapshotMembers()
             StructuredAggregationQuery = new StructuredAggregationQuery
             {
                 StructuredQuery = query.ToStructuredQuery(),
-                Aggregations = { new Aggregation { Alias = "Count", Count = new Count() } }
+                Aggregations = { new Aggregation { Alias = "A1", Count = new Count() } }
             }
         };
         var response = new FakeAggregationQueryStream(new[]
         {
             new RunAggregationQueryResponse { ReadTime = sampleReadTime, Result = new AggregationResult() }
         });
         mock.Configure().RunAggregationQuery(request, Arg.Any<CallSettings>()).Returns(response);
+
         var aggregateQuery = query.Count();
         var snapshot = await aggregateQuery.GetSnapshotAsync();
+
+        Assert.Equal(aggregateQuery, snapshot.Query);
+        Assert.Equal(Timestamp.FromProto(sampleReadTime), snapshot.ReadTime);
+        // We didn't provide a count in the response.
+        Assert.Null(snapshot.Count);
```

Comment:
I think we wrote this case to throw, so it would have consistent behavior with `snapshot.GetValue<long>("Count")`. This might be worth further discussion.

Reply:
This is an unfortunate result of us not meeting when designing Count - the Unity SDK has a non-nullable property; the .NET SDK has a nullable property. I think we should continue in the spirit of that discrepancy rather than fixing it by having a nullable property which never returns null. Happy to discuss further if you feel strongly about it though.

Reply:
I'm fine continuing as is.